alws/alembic/versions/7469773f4f79_add_priority_to_repositories.py

```diff
@@ -0,0 +1,30 @@
+"""add priority to repositories
+
+Revision ID: 7469773f4f79
+Revises: 3da2d1e48185
+Create Date: 2022-02-17 09:36:28.521829
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '7469773f4f79'
+down_revision = '3da2d1e48185'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Below commands doesn't look like auto generated by `alembic`, so u can delete this comment, not critical